src/main/java/org/broadinstitute/hellbender/tools/walkers/mutect/PermutectDatasetEngine.java

```diff
@@ -134,10 +134,12 @@ public void addData(final ReferenceContext ref, final VariantContext vc, Optiona
         final int numAlt = vc.getNAlleles() - 1;
 
 
-        // the variant has already been annotated, so we have POPAF and AD
+        // the variant has already been annotated, so we have POPAF, AD, and in-PON status
         final double[] popafs = VariantContextGetters.getAttributeAsDoubleArray(vc, GATKVCFConstants.POPULATION_AF_KEY);
         //final double[] altPopulationAFs = MathUtils.applyToArray(popafs, x -> Math.pow(10, -x ));
         final double[] tumorLods = Mutect2FilteringEngine.getTumorLogOdds(vc);
+        // the PoN is NOT for calling! We only use it to assign some unlabeled data as artifacts in training
+        final boolean inPon = vc.hasAttribute(GATKVCFConstants.IN_PON_KEY);
```

Comment:
Shouldn't you rename the variable?  It sounds like this no longer means the variant is actually in the PoN (I mean "what PoN?"  Am I right?)

Reply:
done